docs/DeveloperGuide.adoc

```diff
@@ -292,20 +299,52 @@ Refer to the guide <<DevOps#, here>>.
 
 Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (unlikely to have) - `*`
 
-[width="59%",cols="22%,<23%,<25%,<30%",options="header",]
+[width="59%",cols="23%,<23%,<25%,<30%",options="header",]
 |=======================================================================
-|Priority |As a ... |I want to ... |So that I can...
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
+|Priority |As a ... |I want to ... |So that I ...
+|`* * *` |new user |see usage instructions |can refer to instructions when I forget how to use the application
 
-|`* * *` |user |add a new person |
+|`* * *` |user |input the names of items |
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* * *` |user |input the expiry dates of items |
```

Comment:
This is an incomplete user story, can be simple as a "so that I can see the item's expiry date in the system"